reviewing/outdated/src/outdated.ts

```diff
@@ -59,6 +59,22 @@ export async function outdated (
   const currentLockfile = opts.currentLockfile ?? { importers: { [importerId]: {} } }
 
   const outdated: OutdatedPackage[] = []
+  if (typeof opts.manifest?.packageManager === 'string') {
+    const [pkgManager, pkgManagerVersion] = opts.manifest.packageManager.split('@')
```

Comment:
Will we also support updating other package managers? I don't think corepack will even allow to run pnpm, if the packageManager field contains a different package manager. I think we only need to support updating pnpm.

Reply:
registry-mock works as a proxy too. So it should fetch pnpm from the public registry. No need to add pnpm to it. It should work already.